docs/content/ActionList.mdx

```diff
@@ -62,11 +65,36 @@ An `ActionList` is a list of items which can be activated or selected. `ActionLi
 />
 ```
 
-## Component props
+## Example with custom item renderer
+
+```jsx
+<ActionList
+  items={[
+    {
+      text: 'Vanilla link',
+      renderItem: props => <ActionList.Item as="a" href="/about" {...props} />
+    },
+    {
+      text: 'React Router link',
+      renderItem: props => <ActionList.Item as={ReactRouterLikeLink} to="/about" {...props} />
+    },
+    {
+      text: 'NextJS style',
+      renderItem: props => (
+        <NextJSLikeLink href="/about">
+          <ActionList.Item as="a" {...props} />
+        </NextJSLikeLink>
+      )
+    }
+  ]}
+/>
+```
+
+## ActionList props
 
-| Name             | Type                                |      Default      | Description                                                                                                                                             |
-| :--------------- | :---------------------------------- | :---------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| items            | `ItemProps[]`                       |    `undefined`    | Required. A list of item objects conforming to the `ActionList.Item` props interface.                                                                   |
-| renderItem       | `(props: ItemProps) => JSX.Element` | `ActionList.Item` | Optional. If defined, each item in `items` will be passed to this function, allowing for `ActionList`-wide custom item rendering.                       |
-| groupMetadata    | `GroupProps[]`                      |    `undefined`    | Optional. If defined, `ActionList` will group `items` into `ActionList.Group`s separated by `ActionList.Divider` according to their `groupId` property. |
-| showItemDividers | `boolean`                           |      `false`      | Optional. If `true` dividers will be displayed above each `ActionList.Item` which does not follow an `ActionList.Header` or `ActionList.Divider`        |
+| Name             | Type                                |               Default               | Description                                                                                                                                             |
+| :--------------- | :---------------------------------- | :---------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------- |
+| items            | `Array<ItemProps                    | (props: ItemProps) => JSX.Element>` | `undefined`                                                                                                                                             | Required. A list of item objects conforming to the `ActionList.Item` props interface. |
```

Comment:
Looks like the `|` in the type is confusing Markdown: 
![CleanShot 2021-09-29 at 09 05 53](https://user-images.githubusercontent.com/4608155/135306747-5450ca3f-4b9f-471c-b4c9-95619ed8b982.png)